dvc/fs/local.py

```diff
@@ -21,70 +19,42 @@ class LocalFileSystem(BaseFileSystem):
     PARAM_CHECKSUM = "md5"
     PARAM_PATH = "path"
     TRAVERSE_PREFIX_LEN = 2
+    HIDDEN_DIRS = {".git", ".hg", ".dvc"}
```

Comment:
Should this be part of dvcignore or some additional filter? Seems like it doesn't really belong on fs level.